extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -963,36 +1022,38 @@ class MultipleBackendTask extends ReplicateObject {
         log.debug('replicating object tags', {
             entry: sourceEntry.getLogInfo(),
         });
-        const destReq = this.backbeatSource
-            .multipleBackendPutObjectTagging({
-                Bucket: sourceEntry.getBucket(),
-                Key: sourceEntry.getObjectKey(),
-                StorageType: sourceEntry.getReplicationStorageType(),
-                StorageClass: this.site,
-                DataStoreVersionId:
-                    sourceEntry.getReplicationSiteDataStoreVersionId(this.site),
-                Tags: JSON.stringify(sourceEntry.getTags()),
-                SourceBucket: sourceEntry.getBucket(),
-                SourceVersionId: sourceEntry.getVersionId(),
-                ReplicationEndpointSite: this.site,
-            });
-        attachReqUids(destReq, log);
-        const writeStartTime = Date.now();
-        return destReq.send((err, data) => {
-            if (err) {
+        const command = new MultipleBackendPutObjectTaggingCommand({
+            Bucket: sourceEntry.getBucket(),
+            Key: sourceEntry.getObjectKey(),
+            StorageType: sourceEntry.getReplicationStorageType(),
+            StorageClass: this.site,
+            DataStoreVersionId:
+                sourceEntry.getReplicationSiteDataStoreVersionId(this.site),
+            Tags: JSON.stringify(sourceEntry.getTags()),
+            SourceBucket: sourceEntry.getBucket(),
+            SourceVersionId: sourceEntry.getVersionId(),
+            ReplicationEndpointSite: this.site,
+            RequestUids: log.getSerializedUids(),
+        });
+        
+        return this.backbeatSource.send(command)
+            .then(data => {
+                sourceEntry.setReplicationSiteDataStoreVersionId(this.site,
+                    data.versionId);
+                // TODO : review metric
+                // this._publishMetadataWriteMetrics(command.input.Tags, writeStartTime);
```

Comment:
is this about the "size" of the MD ?
even if we keep the TODO to pass the proper size, can be keep some metrics already : so we still
measure/report the number of calls & duration.